requirements.txt

```diff
@@ -1,2 +1 @@
 python-upwork
-smtplib
```

Comment:
I've removed this line because `smtplib` is part of the python standard library.